### PR TITLE
Update integration and query to be able to create SNS event notification for S3

### PR DESCRIPTION
**Pull request by OliviaHY**
Hold merge till https://github.com/app-sre/qontract-schemas/pull/37 is

### CODE REVIEW

**Comment by BumbleFeng** [comment]
Could you separate create SNS topic and SNS event notification for S3 into 2 PR?

**Comment by OliviaHY** [comment]
> Could you separate create SNS topic and SNS event notification for S3 into 2 PR?

Absolutely.

File: reconcile/utils/terrascript_client.py [hunk 1]
```diff
@@ -3754,3 +3800,34 @@ def populate_tf_resource_secrets_manager(self, resource, namespace_info):
 
         for tf_resource in tf_resources:
             self.add_resource(account, tf_resource)
+        
+    def populate_tf_resource_sns(self, resource, namespace_info):
```
Comment (Piojo) [inline reconcile/utils/terrascript_client.py:3804]: I'm pretty sure we can add unit tests for this. :)

**Review by Piojo — CHANGES_REQUESTED** [review]
There are merge conflicts. Please rebase and re-submit.

**Pull request closed without merging**